index.html

```diff
@@ -25,68 +47,160 @@ <h1 id="result"></h1>
     </div>
 
     <script>
-        //Grupy poniedziałkowe oraz wtorkowe będą miały cały trening gier na drugich zajęciach w nieparzyste miesiące (np. 14 i 15 listopada), 
+        //Grupy poniedziałkowe oraz wtorkowe będą miały cały trening gier na drugich zajęciach w nieparzyste miesiące (np. 14 i 15 listopada),
         //a grupy środowe, czwartkowe, piątkowe i sobotnie na drugich zajęciach w parzyste miesiące (np. 8, 9, 10 i 14 grudnia)
 
+        // Wyjątkiem (dyskretnie zawartym w regule powyzej) jest sytuacja gdy pierwszy dany dzień tygodnia w miesiącu jest świętem. Przykładowo 6.01.2025- pierwszy poniedziałęk roku to święto, więc pierwsze zajęcia przypadają na 13.01.2025 i wtedy drugie zajęcia będą 20.01.2025 i to wtedy są gierki. 
+
+
         // Get today's date
         var today = new Date();
-        //today.setDate(today.getDate()+1);
-        var day = today.getDay();
         var date = today.getDate();
         var month = today.getMonth();
         var year = today.getFullYear();
+        const api_url = "https://openholidaysapi.org/PublicHolidays?countryIsoCode=PL&validFrom=" + year + "-01-01&validTo="+ year +"-12-31";
```

Comment:
zmienne jak ta to są z dużej litery ;)

Reply:
oo rzeczywiście, python ze mnie wyszedł.

Reply:
Skoro to const to jeszcze bym dał bardziej specific, że to HOLIDAY_API_URL.